poc/vdaf_mastic.py

```diff
@@ -326,11 +327,12 @@ def helper_proof_share(cls, flp_seed):
         )
 
     @classmethod
-    def joint_rand_part(cls, agg_id, flp_seed, beta_share, nonce):
+    def joint_rand_part(cls, agg_id, flp_seed, beta_share, public_share, nonce):
```

Comment:
For consistency (plus `beta_share` is not rthe right term!)
```suggestion
        def joint_rand_part(cls, agg_id, flp_seed, vidpf_key, vidpf_public_share, nonce):
```